packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -308,23 +303,26 @@ abstract contract MajorityVotingBase is
         if (_isVoteOpen(proposal_)) {
             // Early execution
             return
-                worstCaseSupport(_proposalId) > proposal_.supportThreshold &&
-                participation(_proposalId) >= proposal_.minParticipation;
+                proposal_.configuration.voteMode == VoteMode.EarlyExecution &&
+                worstCaseSupport(_proposalId) > proposal_.configuration.supportThreshold &&
+                participation(_proposalId) >= proposal_.configuration.minParticipation;
         } else {
```

Comment:
Given the return, the `else` is implied. We may flatten the code after the `if()`

Reply:
Amended in 27597a2fa935fb490aa855a3f5aff33373178071